internal/command/commands/service.go

```diff
@@ -105,37 +105,37 @@ func (s *Service) Run(args []string) int {
 			s.Logger.Error(err.Error())
 			return 1
 		}
-		envFileMap := make(map[string]string)
 		r, _ := regexp.Compile(`provisioning-([a-zA-Z]*)\.json`)
-		for _, file := range allFiles {
-			envType := r.FindStringSubmatch(file)
-			if len(envType) > 1 {
-				envFileMap[envType[1]] = file
-			}
-		}
-
 		provisioningConfigMap := make(map[string]interface{})
-		for _, envType := range envTypes.EnvTypes {
-			if envFileMap[envType] == "" {
-				continue
-			}
-			f := filepath.Join(*directoryPath, utils.GetProvisioningFileName(envType))
 
-			data, provisioningFilePath, err := file.FindAndReadAllAllowedFormat(f, []string{".json", ".yml", ".yaml"})
-			// Ignore empty provisioning files
-			if len(data) == 0 {
-				continue
-			}
-			if err != nil {
-				s.Logger.Error(err.Error())
-				return 1
-			}
-			parsedProvisioningConfig, err := utils.ParserYmlOrJson(provisioningFilePath, data)
-			if err != nil {
-				s.Logger.Error(err.Error())
-				return 1
+		for _, fileName := range allFiles {
+			envType := r.FindStringSubmatch(fileName)
+			if len(envType) == 0 {
+				if fileName != "definition.json" {
```

Comment:
here the check is for definition.json, but the message says its provisioning

Reply:
It is to not show warning message for definition.json as it is a valid file